### PR TITLE
[Blog Post] Tables are Hard, Part 2: Building a Simple Data Table in React

### PR DESCRIPTION
**Pull request by NickLanam**
Adds a new component for tabs that this post needed. It's not great for code yet (look at the Markdown to see why), but we can fix that in the future. It serves it purpose for now.

Depends on the [demo diff in pixie-demos](https://github.com/pixie-io/pixie-demos/pull/24) for the embeds to work. Don't merge this until that diff has been merged and its GitHub Pages config is updated to match.

Signed-off-by: Nick Lanam <nlanam@pixielabs.ai>

### CODE REVIEW

**Review by aimichelle — APPROVED** [review]

File: content/blog/tables-are-hard-2/index.md [hunk 1]
```diff
@@ -0,0 +1,825 @@
...
+Let's build a "simple" data table in a web app, starting with the basics.
+
+In the [previous post](https://blog.px.dev/tables-are-hard-1/), we summarized the features expected in a modern data table.
+
+By the end of this tutorial, we'll build a table with filtering, sorting, and basic column controls.
+In the next post, we'll add new requirements and walk through handling them.
+
+This tutorial is aimed at relative beginners to web development, and will keep the set of tools as basic as possible to remain focused. Skip to the end for suggestions for a more serious project.
+
+[Final Code](https://github.com/pixie-io/pixie-demos/react-table-demo) | [Live Demo](https://pixie-io.github.io/pixie-demos/react-table-demo)
```
Comment (htroisi) [inline content/blog/tables-are-hard-2/index.md:20]:
```suggestion
[Final Code](https://github.com/pixie-io/pixie-demos/react-table) | [Live Demo](https://pixie-io.github.io/pixie-demos/react-table)
```

File: content/blog/tables-are-hard-2/index.md [hunk 2]
```diff
@@ -0,0 +1,825 @@
...
+}\n\
+\n\
+```"}
+</tab>
+
+</tabs>
+
+Although column ordering also has a plugin, it doesn't provide props for easy controls like the others. As such, we'll skip it for this tutorial.
+
+Here's what it looks like so far:
```
Comment (htroisi) [inline content/blog/tables-are-hard-2/index.md:708]:
```suggestion
Here's what it looks like so far (click to interact):
```

File: content/blog/tables-are-hard-2/index.md [hunk 3]
```diff
@@ -0,0 +1,825 @@
...
+   ), [numRows, numCols]);\n\
+ \n\
+   return { columns, data };\n\
+ }\n\
+\n\
+```"}
+</tab>
+</tabs>
+
+With this feature, our demo is largely complete (interactive demo):
```
Comment (htroisi) [inline content/blog/tables-are-hard-2/index.md:805]:
```suggestion
With this feature, our demo is largely complete (click to interact):
```

**Review by htroisi — APPROVED** [review]

File: content/blog/tables-are-hard-2/index.md (hunk 1 above)
Comment (NickLanam) [reply content/blog/tables-are-hard-2/index.md]: Oops, nice catch! Done.

File: content/blog/tables-are-hard-2/index.md (hunk 2 above)
Comment (NickLanam) [reply content/blog/tables-are-hard-2/index.md]: Done.

File: content/blog/tables-are-hard-2/index.md (hunk 3 above)
Comment (NickLanam) [reply content/blog/tables-are-hard-2/index.md]: Done.

**Review by aimichelle — APPROVED** [review]

**Pull request merged**